fcrepo-server/src/main/java/org/fcrepo/server/resourceIndex/ResourceIndex.java

```diff
@@ -79,6 +86,18 @@ void modifyObject(DOReader oldReader, DOReader newReader)
     void deleteObject(DOReader oldReader) throws ResourceIndexException;
 
     /**
+     * Export the triples indexed from a given object.
+     * Adding an object is the equivalent of inserting these triples.
+     * Deleting an object is the equivalent of deleting these triples.
+     * Modifying an object is the equivalent of processing the difference
+     * between the export before and after a change.
+     * @param object
+     * @return
+     * @throws ResourceIndexException
+     */
+    List<Triple> exportObject(DOReader object) throws ResourceIndexException;
```

Comment:
It lines up better with the org.trippi.TriplestoreWriter interface that this interface extends.